docs/build/web.md

```diff
@@ -75,14 +89,7 @@ in `<ORT_ROOT>/`, run one of the following commands to build WebAssembly:
 ./build.sh --config Release --build_wasm --skip_tests --disable_wasm_exception_catching --disable_rtti
 ```
 
-ONNX Runtime WebAssembly can be built with or without multi-thread/SIMD support, specified by appending the following flags.
-
-| build flag              | usage                           |
-| ----------------------- | ------------------------------- |
-| `--enable_wasm_threads` | build with multi-thread support |
-| `--enable_wasm_simd`    | build with SIMD support         |
-
-To get all build artifacts of ONNX Runtime WebAssembly, it needs 4 times of build with the combinations of ON/OFF of the 2 flags. A full list of build artifacts are as below:
+A full list of build artifacts are as below:
```

Comment:
I like it to move to top of the introduction of flags --enable_wasm_threads and --enable_wasm_simd.

however, this was connected with the artifacts table. do you think we should also move this table to above?

Reply:
I feel like this way the description flows better. we define what configurations of WASM artifacts are needed and how it's controlled --> We explain how to generate the artifacts --> listing the artifacts we'll need to complete the build. the flow is "Why" --> "How" --> "What", which is the end result. I guess moving it up to be "Why" --> "What" --> "How" will also work, for me finishing with "what to expect" from this step feels more in place but I am OK either way :)